### PR TITLE
Update deps based on github security audit

### PR DESCRIPTION
**Pull request by jnatalzia**
### This PR will...
Update deps to remove security issues

*NOTE:* The following vulnerabilities still exists. This is because:
* in order to upgrade the version of `braces` required, it would require breaking changes to our stylelint rules. 
* jsdoc does not have a version that resolves the `marked` vulnerability

Given they are `low` and `medium` vulnerabilities, I do not see it as a blocker to merging this given the coverage elsewhere

```
yarn audit v1.19.1
┌───────────────┬──────────────────────────────────────────────────────┐
│ low           │ Regular Expression Denial of Service                 │
├───────────────┼──────────────────────────────────────────────────────┤
│ Package       │ braces                                               │
├───────────────┼──────────────────────────────────────────────────────┤
│ Patched in    │ >=2.3.1                                              │
├───────────────┼──────────────────────────────────────────────────────┤
│ Dependency of │ stylelint                                            │
├───────────────┼──────────────────────────────────────────────────────┤
│ Path          │ stylelint > micromatch > braces                      │
├───────────────┼──────────────────────────────────────────────────────┤
│ More info     │ https://www.npmjs.com/advisories/786                 │
└───────────────┴──────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────┐
│ moderate      │ Regular Expression Denial of Service                 │
├───────────────┼──────────────────────────────────────────────────────┤
│ Package       │ marked                                               │
├───────────────┼──────────────────────────────────────────────────────┤
│ Patched in    │ >=0.6.2                                              │
├───────────────┼──────────────────────────────────────────────────────┤
│ Dependency of │ jsdoc                                                │
├───────────────┼──────────────────────────────────────────────────────┤
│ Path          │ jsdoc > marked                                       │
├───────────────┼──────────────────────────────────────────────────────┤
│ More info     │ https://www.npmjs.com/advisories/812                 │
└───────────────┴──────────────────────────────────────────────────────┘
2 vulnerabilities found - Packages audited: 13010
Severity: 1 Low | 1 Moderate
```

### Why is this Pull Request needed?
We have vulnerabilities

### Are there any points in the code the reviewer needs to double check?
Do the builds pass?

### Are there any Pull Requests open in other repos which need to be merged with this?
Nah

#### Addresses Issue(s):

JW8-10677

### Checklist
- [ ] Jenkins builds and unit tests are passing
- [ ] I have reviewed the automated results

### CODE REVIEW

**Comment by jnatalzia** [comment]
test this please

**Comment by jnatalzia** [comment]
it's almost certainly karma 4 and prob config related. great catch, sorry for all the back and forth on this one

**Comment by jnatalzia** [comment]
test this please

**Comment by robwalch** [comment]
@jnatalzia np. Other builds were failing because of the BrowserStack authkey change, but now that that is fixed, only this build is failing with a basic auth error. The thing is there should be no basic auth on these tests. The last unit test build "jenkins-jw7-opensource-unit-tests-5298" didn't even get logged on Browser stack. It aborted with a timeout;
https://jenkins.jwplayer.com/view/JW7/job/jw7-opensource-unit-tests/5298/console

Compare to the one before that:
https://automate.browserstack.com/dashboard/v2/builds/9a69c1dbf9bf251d92b211ac22c69a175fea8d9b/sessions/cbb164c218f0b5252495c25a270809af3cade8ca
https://jenkins.jwplayer.com/view/JW7/job/jw7-opensource-unit-tests/5297/console

browserstack local is required. The issue is probably related to configuring that properly with karma 4.

File: package.json [hunk 1]
```diff
@@ -23,18 +23,18 @@
...
-    "grunt-karma": "2.0.0",
+    "grunt-contrib-watch": "1.1.0",
+    "grunt-karma": "3.0.0",
     "grunt-postcss": "0.9.0",
     "intersection-observer": "0.5.0",
     "jsdoc": "3.5.5",
-    "karma": "4.0.0",
-    "karma-browserstack-launcher": "1.3.0",
+    "karma": "4.0.1",
+    "karma-browserstack-launcher": "1.5.1",
```
Comment (robwalch) [inline package.json]: See https://github.com/karma-runner/karma-browserstack-launcher/issues/152

**Pull request merged**